contrib/cirrus/setup_environment.sh

```diff
@@ -46,6 +46,9 @@ case "${OS_RELEASE_ID}" in
 
         workaround_bfq_bug
 
+	# HACK: Need Conmon 2.0.17, currently in updates-testing on F31.
+	dnf update -y --enablerepo=updates-testing conmon
```

Comment:
Any way we can push this through to stable?  I'd like to get the hack part out of this.

Reply:
It's in the process of pushing, but I don't know how long it takes to make it into the repos